### ISSUE TITLE
Always runs rspec for ruby files

### ISSUE THREAD

**Issue by Linuus** [syl20bnr/spacemacs#6173]
#### Description :octocat:
When running a test in a ruby file it always uses rspec, even if I explicitly set it to use ruby-test in a `.dir-locals.el` file.

#### Reproduction guide :beetle:
- Start Emacs
- Create a `.dir-locals.el` file with the content `((ruby-mode (ruby-test-runner . ruby-test)))`
- Go to `test/some/file_test.rb`
- Press `SPC m t t` on a test

*Observed behaviour:* :eyes:
It runs rspec even though `ruby-test-runner` is set to `ruby-test`.
`SPC m t t` is mapped explicitly to rspec so...

*Expected behaviour:* :broken_heart:
It runs ruby-test

#### System Info :computer:
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.20
- Spacemacs branch: develop (rev. 60f5a3a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion emacs-lisp elixir
                 (elfeed :variables elfeed-feeds
                         '(("http://sachachua.com/blog/category/emacs/feed/" emacs)
                           ("http://learningelixir.joekain.com/full-feed.xml" elixir)
                           ("http://blog.plataformatec.com.br/feed/" elixir)
                           ("http://planet.emacsen.org/atom.xml" emacs)))
                 erc erlang git javascript latex osx dash html markdown ruby ruby-on-rails org yaml
                 (shell :variables shell-default-shell 'eshell shell-default-height 30 shell-default-position 'bottom)
                 syntax-checking version-control themes-megapack org-page)
```

#### Backtrace :paw_prints:
```
Have no backtrace
```

The relevant code is here: https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Blang/ruby/packages.el#L209-L228

**Comment by syl20bnr** [comment]
It should be fixed in develop now :-)

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
